src/modules/endpoint/sagas/__tests__/fetchResourceCache.spec.js

```diff
@@ -44,21 +43,33 @@ describe('fetchResourceCache', () => {
 
     const fetchedResources = await dataContainer.getResourceCache(city, language)
 
-    const lessStrictExpected = transform(resources, (result, value, path) => {
-      forEach(value, (value, url) => {
-        result[path][url].lastUpdate = expect.any(Moment)
-      })
-      return result
-    }, resources)
+    if (fetchedResources.errorMessage !== undefined) {
+      throw new Error('getResourceCache threw an error!')
+    }
+
+    const fetchedCount = {
+      ...fetchedResources['/augsburg/de/category_0'],
+      ...fetchedResources['/augsburg/de/category_0/category_0']
+    }
+
+    const expectedCount = {
+      ...resources['/augsburg/de/category_0'],
+      ...resources['/augsburg/de/category_0/category_0']
+    }
+
+    expect(Object.keys(fetchedCount)).toHaveLength(
+      Object.keys(expectedCount).length - 1 /* The first url is excluded because the
+                                               FetcherModule mock produced an error for it */
```

Comment:
You cannot *really* say that it's the first url, since Object.keys' order is arbitrary. But you can say, that there's one entry that the FetcherModule produced an error for.

Reply:
Forgot to adjust this. Thanks!